ci/envs/311-latest.yaml

```diff
@@ -10,16 +10,13 @@ dependencies:
   - libpysal
   - py-opencv
   # tests
-  - nose
-  - nose-exclude
-  - coverage
-  - coveralls
   - scikit-learn
   - shapely
   - pygeos
```

Comment:
Is `pygeos` needed?

Reply:
no clue, it was in the list when I came to it :)

Reply:
(same, just commenting more like a stream of consciousness so I don't forget)

Reply:
https://github.com/pysal/pointpats/blob/1a35321aeb4afd52404817ae904b0d53a3ac1d9b/pointpats/geometry.py#L223

Reply:
Yep, was just about to post that.